code/modules/nano/interaction/default.dm

```diff
@@ -102,3 +102,10 @@
 		. = min(., shared_living_nano_distance(src_object))
 		if(. == STATUS_UPDATE && (TK in mutations))	// If we have telekinesis and remain close enough, allow interaction.
 			return STATUS_INTERACTIVE
+
+/mob/living/carbon/alien/default_can_use_topic(var/src_object)
+	. = shared_nano_interaction(src_object)
+	if(. != STATUS_CLOSE)
+		. = min(., shared_living_nano_distance(src_object))
+		if(!IsAdvancedToolUser())
+			. = STATUS_UPDATE
```

Comment:
I'm not sure whether we should go with STATUS_UPDATE or STATUS_CLOSE here, though I'm leaning towards STATUS_CLOSE. It depends on whether or not we want Xenos to be able to still read monitors and other things when they aren't an advanced tool user.